spec/spec_helper.cr

```diff
@@ -7,10 +6,8 @@ Granite::Adapters << Granite::Adapter::Mysql.new({name: "mysql", url: ENV["MYSQL
 Granite::Adapters << Granite::Adapter::Pg.new({name: "pg", url: ENV["PG_DATABASE_URL"]})
```

Comment:
This isn't going to work since we establish a connection when this get's instantiated.  We should lazy load the connection.  @Thellior Can you fix that?